langchain/callbacks/manager.py

```diff
@@ -629,11 +631,10 @@ def _configure(
         )
         [callback_manager.add_handler(handler, False) for handler in local_handlers_]
 
+    if not callback_manager:
```

Comment:
this change is a deviation from JS, where callback manager remains optional. Should I do this there too?